load_site.py

```diff
@@ -0,0 +1,27 @@
+import asyncio
+import aiohttp
+import time
+
+
```

Comment:
Передавать url в функции как параметр, а брать из константы, user input или из других модулей.
```suggestion
URLS = ['https://ya.ru', 'https://google.com']```

Reply:
Only now I noticed Github has a common confirming button that needs to be clicked before posting all the comments...

Reply:
No problem, thank you